index.js

```diff
@@ -17,6 +17,15 @@ const bot = controller.spawn({
   token: slackBotToken
 })
 
+const redisUrl = process.env.REDIS_URL;
+
+if (!redisUrl) {
```

Comment:
@yuya-takeyama 
I was thinking that it would acceptable to let using Redis **optional** in order to keep this bot requirement minimum. If it would get more feature to use it in future, it will be **required** at that time.

What do you think?